app/javascript/packages/analytics/index.ts

```diff
@@ -15,26 +15,24 @@ interface NewRelicGlobals {
  *
  * @return Promise resolving once event has been logged.
  */
-export async function trackEvent(event: string, payload: object = {}): Promise<void> {
+export async function trackEvent(event: string, payload?: object): Promise<void> {
   const endpoint = getConfigValue('analyticsEndpoint');
   if (!endpoint) {
     return;
   }
 
-  try {
-    await window.fetch(endpoint, {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({ event, payload }),
-    });
-  } catch (error) {
+  await fetch(endpoint, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ event, payload }),
```

Comment:
what if we used `||` to do what the default value did? it's still an overall character reduction right?
```suggestion
    body: JSON.stringify({ event, payload: payload || {} }),
```

Reply:
> what if we used `||` to do what the default value did? it's still an overall character reduction right?

Yeah this would technically work and be reasonably effective at reducing overall size. To my other comment though, I guess I don't see the issue with omitting the payload if we have nothing to send.